### PR TITLE
Cassandra 2.2.4

### PR DESCRIPTION
**Pull request by djchen**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Install is failing: http://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1143/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install_homebrew_versions_cassandra22/

Audit is failing too: http://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1143/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_homebrew_versions_cassandra22___strict___online/

Also, can you squash this to a single commit? Thanks!

**Comment by djchen** [comment]
Yeah sorry about that, I'm still working on it. Trying to figure out why `https://archive.apache.org/dist/cassandra/2.2.3/apache-cassandra-2.2.3-bin.tar.gz` fails on wget but downloads ok on the browser.

**Comment by djchen** [comment]
@mikemcquaid Do you have any docs or pointers on whats best to add for the test block?

**Comment by mikemcquaid** [comment]
@djchen Something like what `jenkins.rb` does in Homebrew core would be good. If you're feeling extra-kind a PR to add that to the Homebrew core Cassandra formula would be great too.

**Comment by djchen** [comment]
Thanks for the pointers. I looked at the possibility of the doing the same thing with Cassandra and did not see a good way of telling it to use the tmp dir for writing data to. I've added a very basic `test do` block for now.

File: cassandra22.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
+class Cassandra22 < Formula
+  desc "Eventually consistent, distributed key-value store"
```
Comment (mikemcquaid) [inline cassandra22.rb]: Can you make this shorter: http://bot.brew.sh/job/Homebrew%20Versions%20Pull%20Requests/1152/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_homebrew_versions_cassandra22___strict___online/

**Comment by mikemcquaid** [comment]
@djchen That's fine for now. Mind making a pull request with the test block to Homebrew's main repository for `cassandra.rb` too? Thanks!

**Comment by djchen** [comment]
Will do on the PR to the main repo. Desc updated, should be under 80 chars now.

**Comment by djchen** [comment]
@mikemcquaid Are we good to merge this?

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

Apologies for the delay here!

**Pull request closed without merging**